docs/src/lib/TokenPresenter.jsx

```diff
@@ -24,7 +24,7 @@ export function TokenPresenter( { token } ) {
 	const name = token.name;
 
 	if ( isPropertyToken( name ) ) {
-		return <Default token={ token } />;
```

Comment:
The import of `Default` needs to be removed. I was doing something based on this branch and saw storybook doesn't build.

Reply:
Thanks. Proof that bugs really solve themself over night.